lib/features/family/app/family_pages.dart

```diff
@@ -64,6 +64,10 @@ enum FamilyPages {
       path: 'choose-amount-slider-goal', name: 'CHOOSE_AMOUNT_SLIDER_GOAL'),
 
   //generosity challenge
+  generosityChallengeRedirect(
+    path: '/generosity-challenge-redirect',
```

Comment:
Does this need to be a root path or should it be without the slash?

Reply:
Oh right, ill put it at the same level as registrationUs